.gitignore

```diff
@@ -66,6 +66,9 @@ nosetests.xml
 coverage.xml
 coverage/
 
+# Cypress testing
+cypress/videos/
+
```

Comment:
i'd also include the other cypress folders of the artifacts that could get generated during testing.

`
cypress/downloads/
cypress/screenshots/
`

Reply:
good catch.